package.json

```diff
@@ -37,8 +37,8 @@
     "yaml-front-matter": "^4.1.1"
   },
   "scripts": {
-    "spin": "spin watch --file spin.toml --quiet --env PREVIEW_MODE=$PREVIEW_MODE -e BASE_URL=http://localhost:3000",
-    "bundle-scripts": "parcel build static/js/src/main.js --dist-dir static/js --no-source-maps",
+    "spin": "nodemon --watch content --watch static --watch templates --watch spin-up-hub --ext html,md,rhai,hbs,css,js --verbose --legacy-watch --signal SIGINT --exec 'npm run build-index && spin up --file spin.toml --quiet --env PREVIEW_MODE=$PREVIEW_MODE'",
+    "bundle-scripts": "parcel watch static/js/src/main.js --dist-dir static/js --no-source-maps",
```

Comment:
Double-checking: was this meant to change to `watch` here as well?

Reply:
The `watch` on bundle-scripts was an artifact of my testing the feedback changes. It needs to be `build` when we eventually want to move the building of CSS and JS to the CI/CD pipeline as well.  This is because `parcel watch` by default looks for changes in its input files and never terminates. 
we might just add new npm script commands for when that happens.